glfw/src/clipboard.zig

```diff
@@ -39,10 +39,10 @@ pub inline fn setClipboardString(value: [*c]const u8) Error!void {
 /// @thread_safety This function must only be called from the main thread.
 ///
 /// see also: clipboard, glfwSetClipboardString
-pub inline fn getClipboardString() Error![*c]const u8 {
+pub inline fn getClipboardString() Error![:0]const u8 {
     const value = c.glfwGetClipboardString(null);
     try getError();
-    return value;
+    return std.mem.span(value);
```

Comment:
If we do need the null checks, do we not also need one here?

Reply:
We don't, because that function will never return null according to GLFW docs

Reply:
Ah, okay, I see now it only returns NULL if an error occurred. Makes sense!